config.template

```diff
@@ -68,7 +68,11 @@ rclone_options="--buffer-size 500M --checkers 16"
 
 rclone_beta="0" # Set this to "1" if you want to live dangerous and try out rclone latest beta.
 
-rclone_mount_options="${rclone_options} --allow-non-empty --allow-other --max-read-ahead 30G"
+if [${pool_choice}] = "1" ]; then
```

Comment:
Typo issue @Dulanic : 

- if [${pool_choice}] = "1" ]; then 
+ if [ "${pool_choice}" = "1" ]; then

Reply:
You should include this modification to the mount.remote script